planning/behavior_path_planner/test/test_behavior_path_planner_node_interface.cpp

```diff
@@ -89,6 +93,20 @@ void publishMandatoryTopics(
     test_target_node, "behavior_path_planner/input/lateral_offset");
 }
 
+bool isExpectedModuleRunning(
+  std::shared_ptr<BehaviorPathPlannerNode> test_target_node, std::string module_name)
+{
+  // check if the expected module is working
+  auto execution_requested_modules_name = test_target_node->getRunningModules();
+  for (const auto & execution_requested_module_name : execution_requested_modules_name) {
+    if (execution_requested_module_name == module_name) {
+      return true;
+    }
+  }
+
+  return false;
+}
```

Comment:
@kyoichi-sugahara 

Sorry for highjacking the review.
Just a suggestion, how about using `std::any_of` in these lines? It will make the intent more clearer.

Reply:
@zulfaqar-azmi-t4 
Thank you for good review and highjacking :grin: 
I wil apply your suggestion.